src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Models/Properties/Resources.Designer.cs

```diff
@@ -70,6 +70,15 @@ public class Resources {
         }
         
         /// <summary>
+        ///   Looks up a localized string similar to RetentionCount in Weeks should be 1-520, in Months should be 1-120 and Years should be 1-10.
```

Comment:
1. No need to capitalize in the middle of the sentence
2. Please don't use class names such as "RetentionCount" in the user facing strings. Consider using "Retention count"